src/WCSimEventAction.cc

```diff
@@ -227,10 +227,10 @@ void WCSimEventAction::EndOfEventAction(const G4Event* evt)
     G4String name =   WCIDCollectionName;
     G4int collectionID = SDman->GetCollectionID(name);
     if(collectionID>-1) WCHC = (WCSimWCHitsCollection*)HCE->GetHC(collectionID);
-    G4cout << G4endl;
-    G4cout << "WCSimEventAction::EndOfEventAction ☆ (WCSimWCHitsCollection*)" << WCIDCollectionName
-           << " has " << WCHC->entries() << " entries" << G4endl;
-    G4cout << G4endl;
+    //G4cout << G4endl;
```

Comment:
Is this part of the code necessary for your radioactivity simulation? If not, restore default.

Reply:
Done, I kept only the modification needed for relative PMT hit time (which is optionnal via DAQ messenger) and about the overwrite of the tree.